sceptre/scipool/config/bmgfki/sc-product-ec2-linux-docker.yaml

```diff
@@ -58,3 +58,7 @@ sceptre_user_data:
       Info:
         LoadTemplateFromURL: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.4.0/ec2/sc-ec2-linux-docker.yaml'
       Name: 'v1.4.0'
+    - Description: 'Update Amazon Linux AMI {{ range(1, 10000) | random }}'
```

Comment:
need to remove `{{ range(1, 10000) | random }}` from previous version

Reply:
thanks